### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by RyanYoung25**
It's a fix to make it so we can deploy and use the AWS server we've been calling prod. The root of the issue lies somewhere in between a strange configuration and python version nonsense.  None the less this fix will let us keep working until we get the new server up and running.

### CODE REVIEW

File: Backend/updateBackendEnv.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 pip install virtualenv
-virtualenv env
+virtualenv -p python2.6 env
```
Comment (aaronmsegal) [inline Backend/updateBackendEnv.sh]: Sorry if this is a dumb question, but why are we using 2.6 instead of 2.7?

Comment (RyanYoung25) [inline Backend/updateBackendEnv.sh]: That's the root of this whole problem. The mod_wsgi on the apache server that's launching this was compiled with 2.6 which is why our app has always been running using the python2.6 interpreter (Remember that problem with the set syntax?). The server currently has a smattering of tools for python2.6 and python2.7 and it is hard to know which ones are being used when and where. By explicitly forcing the virtualenv on the server to create a python2.6 virtualenv the right tools are being used for everything (by tools I mean things like pip) and they mesh correctly with the mod_wsgi. 

The goal is that once we get the new server up and running (Configured correctly with python2.7) we will be able to switch over and everything will be running 2.7.

**Comment by RyanYoung25** [comment]
It's still good to do a code review, especially to make sure that everyone is on the same page and understands why these changes work.

**Comment by asp78** [comment]
It's a two-line change that is holding up further progress. If people want to see the changes they can look at them after its merged.

**Comment by RyanYoung25** [comment]
Let's give Micah 15 min

**Pull request merged**